### PR TITLE
Improvements to Sass/CSS, use normalize.css, basic fixes for small viewports

### PR DESCRIPTION
**Pull request by georgecrawford**
Smallscreen header:

Smallscreen footer:

Largescreen footer:

### CODE REVIEW

File: app/assets/stylesheets/_variables.scss [hunk 1]
```diff
@@ -1,7 +1,9 @@
 $base-font-size: 18px;
 $base-line-height: 1.6;
-$computed-line-height: $base-line-height * $base-font-size;
```
Comment (aspiers) [inline app/assets/stylesheets/_variables.scss]: We don't need this for anything?

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -66,7 +66,6 @@ p.choose_party {
   background-color: white;
 
   .user {
-    display: inline-block;
```
Comment (aspiers) [inline app/assets/stylesheets/home.scss]: I guess these changes are deliberate but not sure how they relate to rationalising variables - were they supposed to be in a different commit?

File: Gemfile [hunk 3]
```diff
@@ -39,6 +39,7 @@ gem 'omniauth-rails_csrf_protection', '~> 0.1'
 gem 'haml-rails'
 gem 'addressable'
 gem 'normalize-rails'
+gem "autoprefixer-rails"
```
Comment (aspiers) [inline Gemfile]: According to https://github.com/ai/autoprefixer-rails#ruby-on-rails we need to run `rake tmp:clear` after introducing this - is that right?  If so would be helpful to mention this kind of thing in the commit message and on Slack to avoid people getting caught out when they `git pull`.

**Comment by aspiers** [comment]
Addresses #5, at least partially.

**Comment by aspiers** [comment]
I wrote:
> So I'm going to assign that to you for now and mark it in progress, to avoid anyone else taking it on without realising you're working on it.

Oh crap, I can't until you're added to the dev team :-(

**Comment by aspiers** [comment]
Tested locally and confirmed this is a gigantic improvement for mobile devices :-) :+1:

File: app/assets/stylesheets/_variables.scss (hunk 1 above)
Comment (georgecrawford) [reply app/assets/stylesheets/_variables.scss]: Nope, it wasn't used.

File: app/assets/stylesheets/home.scss (hunk 2 above)
Comment (georgecrawford) [reply app/assets/stylesheets/home.scss]: Sorry, my commits weren't brilliantly distinct. I will try and describe things better in future.

File: Gemfile (hunk 3 above)
Comment (georgecrawford) [reply Gemfile]: Thanks. I missed that. I'll ping Slack.
